### ISSUE TITLE
You have created a new client application that uses libraries for user authentication or authorization that will soon be deprecated

### ISSUE THREAD

**Issue by wlavarin34** [devboldly/react-analytics-charts#9]
I keep getting this error in my developer's console. If anyone has faced this issue please let me know!

Error Details: 

"You have created a new client application that uses libraries for user authentication or authorization that will soon be deprecated. New clients must use the new libraries instead; existing clients must also migrate before these libraries are deprecated. See the [Migration Guide](https://developers.google.com/identity/gsi/web/guides/gis-migration) for more information."

**Comment by sudhakarchola** [comment]
Yes, this is happening with recently created clients..Any solution for this?
